tests/e2e/transaction/helper.ts

```diff
@@ -116,15 +116,17 @@ export const rawTransactionMultiAgentHelper = async (
   secondarySignerAccounts: Array<Account>,
   feePayerAccount?: Account,
```

Comment:
should we change it to be a boolean flag?

Reply:
Nah lets not do that, we still need the feed payer account to actually sign the transaction.